std_msgs/Header header

sensor_msgs/Image image
sensor_msgs/PointCloud2 point_cloud

string class_label
geometry_msgs/Pose pose
